components/partials/ArticleComments.vue

```diff
@@ -1,30 +1,233 @@
 <template>
   <section>
-    <h3>11 komentarzy</h3>
+    <h3>Liczba komentarzy</h3>
     <form action="">
       <input type="text">
     </form>
-    <article>
-      <span>Comment 1</span>
-      <section>
-        <span>2 Replies</span>
-        <article>Reply 1</article>
-        <article>Reply 2</article>
-      </section>
-    </article>
-    <article>Comment 2</article>
-    <article>Comment 3</article>
+      <article-comment v-for="(element, index) in comments" :key="index"
+                       :author="element.author.fullname"
+                       :avatar-url="element.author.avatar.url"
+                       :created-at="element.createdAt"
+                       :content="element.text"
+                       :replies-amount="element.replies.length"
+                       :parent-id="element.id"
+                       :is-parent="true"
+                       @clicked-load-replies="getReplies">
+
+        <div v-if="element.isRepliesExpanded">
+          <article-comment v-for="(element, index) in element.replies" :key="index"
+                           :author="element.author.fullname"
+                           :avatar-url="element.author.avatar.url"
+                           :created-at="element.createdAt"
+                           :content="element.text"
+                           :is-parent="false">
+          </article-comment>
+        </div>
+      </article-comment>
+
+      <knit-button v-if="commentsAmount < totalItems" @click.native="onLoadCommentsClick()" 
+                   :buttonClasses="['knit-button--big', 'article-comments__load-button']">Załaduj więcej</knit-button>
   </section>
 </template>
 
 <script>
+  import ArticleComment from '~/components/partials/ArticleComment.vue'
+  import KnitButton from '~/components/KnitButton.vue'
+  import KnitService from '~/services/knitService'
+  import _ from 'lodash'
+
   export default {
+    data () {
+      return {
+        limit: 2,
+        comments: [],
+        page: 0,
+        totalItems: 0
+      }
+    },
+    components: {
+      ArticleComment,
+      KnitButton
+    },
+    props: {
+      articleId: {
+        type: String,
+        required: true
+      }
+    },
+    computed: {
+      commentsAmount () {
+        return this.comments.length
+      }
+    },
+    methods: {
+      onLoadCommentsClick () {
+        document.activeElement.blur()
+        this.getCommentsPage()
+      },
+      getCommentsPage () {
+        const qsObject = {
+          group: ['UserReadLess'],
+          limit: this.limit,
+          'article.id': this.articleId,
+          'order[createdAt]': 'DESC'
+        }
+
+        if (!_.isEmpty(this.comments)) {
+          const date = this.comments[this.commentsAmount - 1].createdAt
+          _.assign(qsObject, { 'createdAt[strictly_before]': date })
+        }
+
+        return KnitService.getCollection(this.$axios, 'comments', qsObject)
```

Comment:
I think these operations should be done via store, but that can be refactored later.

Reply:
We have discussed this and decided not to use the store for comments because this is not what user needs to see first and fast.

Reply:
I think you misunderstood, I'm not saying to use SSR here, but get comments via store, so later when for example user add comments, or goes back to an article from different route, don't need to fetch all data again.

Reply:
What about hot discussions ongoing in article? In your approach user won't see new ones when goes back.

Reply:
@knit-pk/web what do you think?

Reply:
@k911 We did discuss that issue (you were absent when we're talking about it). 
The whole team agrees that comments section should not be in store. We can discuss it one more time one day in the far future and decide which approach is better.